pkg/scheduler/framework/interface_test.go

```diff
@@ -87,6 +87,66 @@ func TestStatus(t *testing.T) {
 	}
 }
 
+func TestIsWaitStatus(t *testing.T) {
+	tests := []struct {
+		name            string
+		status          *Status
+		expectedCode    Code
+		expectedMessage string
+		expectedIsWait  bool
+		expectedAsError error
+	}{
+		{
+			name:            "wait status",
+			status:          NewStatus(Wait, ""),
+			expectedCode:    Wait,
+			expectedMessage: "",
+			expectedIsWait:  true,
+			expectedAsError: nil,
+		},
+		{
+			name:            "error status",
+			status:          NewStatus(Error, "unknown error"),
+			expectedCode:    Error,
+			expectedMessage: "unknown error",
+			expectedIsWait:  false,
+			expectedAsError: errors.New("unknown error"),
+		},
+		{
+			name:            "nil status",
+			status:          nil,
+			expectedCode:    Wait,
```

Comment:
expectedCode should be Success. And expectedIsWait should be false.

Reply:
thanks for pointing that out.
i will fix it👍